pkg/fetch/redhat/appstream-lifecycle/appstream-lifecycle_test.go

```diff
@@ -0,0 +1,88 @@
+package appstreamlifecycle_test
+
+import (
+	"fmt"
+	"io/fs"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	lifecycle "github.com/MaineK00n/vuls-data-update/pkg/fetch/redhat/appstream-lifecycle"
+)
+
+func TestFetch(t *testing.T) {
+	tests := []struct {
+		name     string
+		testdata string
+		hasError bool
+	}{
+		{
+			name:     "happy",
+			testdata: "testdata/fixtures",
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+				if r.URL.Path != "/support/policy/updates/rhel-app-streams-life-cycle" {
+					http.NotFound(w, r)
+					return
+				}
+				if _, err := os.Stat(filepath.Join(tt.testdata, "rhel-app-streams-life-cycle.html")); err != nil {
+					http.NotFound(w, r)
+					return
+				}
+
+				http.ServeFile(w, r, filepath.Join(tt.testdata, "rhel-app-streams-life-cycle.html"))
```

Comment:
```suggestion
				switch r.URL.Path {
				case "/support/policy/updates/rhel-app-streams-life-cycle":
					http.ServeFile(w, r, filepath.Join(tt.testdata, "rhel-app-streams-life-cycle.html"))
				default:
					http.NotFound(w, r)
				}
```

Reply:
done in https://github.com/MaineK00n/vuls-data-update/pull/577/commits/e33e665bfc4541e0aaaf184d937799b47eeec74d